package.json

```diff
@@ -1095,6 +1110,7 @@
         "cross-spawn": "^6.0.5",
         "del": "^3.0.0",
         "fs-extra": "^8.1.0",
+        "got": "^11.5.1",
```

Comment:
Looks like we currently use the `request` package. Why are we introducing another package with similar responsibilities?

Reply:
Looks like `request` is deprecated: https://www.npmjs.com/package/got ; I'll see if it's worth moving off of that to this.

Reply:
What if we prioritize a migration from `request` separate from this change, and use request for the time being? It looks like it was deprecated six months ago, but I don't imagine a migration is necessary for this PR.